src/Cms/FilePicker.php

```diff
@@ -41,13 +41,14 @@ public function items()
 		$model = $this->options['model'];
 
 		// find the right default query
-		if (empty($this->options['query']) === false) {
-			$query = $this->options['query'];
-		} elseif ($model instanceof File) {
-			$query = 'file.siblings';
-		} else {
-			$query = $model::CLASS_ALIAS . '.files';
-		}
+		$query = match (true) {
+			empty($this->options['query']) === false
+				=> $this->options['query'],
+			$model instanceof File
+				=> 'file.siblings',
+			default
+			=> $model::CLASS_ALIAS . '.files'
+		};
```

Comment:
Does this instance of `match` really help? It's even one extra line of code and TBH I find it harder to read than `if elseif else` (considering that each `if` uses a completely different condition).

Reply:
I think it does as it makes clear from the start that it's all about assigning different values to `$query`. In the `if else if else` statement, I need to look into each statement to see whether it also assigns `$query` or something else is done in there.

Reply:
I agree with @distantnative. I think the nicest part for me about match is that I can quickly read it like a return switch once I understood it. It removes all those weird if else statements where you need to define a variable first just to replace the content in the if/else statement afterwards.